src/main/kotlin/com/github/vacxe/googleplaycli/PlayStoreCli.kt

```diff
@@ -21,7 +10,7 @@ import com.google.auth.oauth2.ServiceAccountCredentials
 import java.io.File
 import java.io.FileInputStream
 
-class PlayStoreCli :
+class PlayStoreCli(serviceAccountJson: File, appName: String) :
```

Comment:
Could u revert these changes? For future compatibility with `x509` certificate auth

Reply:
done